Makefile

```diff
@@ -148,12 +164,18 @@ man_pages: man_page
 	./man_page ./man_pages
 	cd man_pages; for i in wwctl*1; do echo "Compressing manpage: $$i"; gzip --force $$i; done
 
+config_defaults:
```

Comment:
@jcsiadal What does config_defaults actually do? I don't see it referenced anywhere.

Reply:
It's in the last line of the first PR comment. Binary for testing that warewulf.conf configuration defaults are being generated as expected. It only works in the build directory. It's not needed, but it's useful for testing changes to the struct.

Reply:
Are you referring to the warewulfconf structure?

Reply:
Yes. I'm referring here to the internal struct used to store unmarshalled YAML, and making sure the other struct of defaults are being applied correctly. Longer-term, I'd like to roll the code into wwsh, so it can generate a clean config with defaults, or clean up an existing configuration file.